assets/src/pages/room/components/StreamPlayer/MediaPlayer.tsx

```diff
@@ -1,38 +1,84 @@
-import React, { RefObject, useEffect, useRef } from "react";
+import React, { RefObject, useEffect, useRef, useState } from "react";
 import clsx from "clsx";
 
 export interface Props {
+  cameraOffImage: JSX.Element | null;
   flipHorizontally?: boolean;
   videoStream?: MediaStream;
   audioStream?: MediaStream;
   playAudio?: boolean;
+  screenShare?: boolean;
 }
 
-const MediaPlayer: React.FC<Props> = ({ videoStream, audioStream, flipHorizontally, playAudio }: Props) => {
+const MediaPlayer: React.FC<Props> = ({
+  videoStream,
+  audioStream,
+  flipHorizontally,
+  playAudio,
+  cameraOffImage,
+  screenShare,
```

Comment:
please rename screenShare to something like"allowFillContent" or "blockFillContetn" to make it more generic